mongoDB/index.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-console */
```

Comment:
if this is temporary then i suggest adding a 

`//FIXME: remove lint disable` so things like this dont linger